basicsr/archs/hifacegan_arch.py

```diff
@@ -0,0 +1,265 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+# 20210519 [Lotayou]: Currently Pytorch official SyncBatchNorm only works under DDP training.
```

Comment:
`20210519 [Lotayou]:`   is not necessary, because git will track the date and the author